javascript/javascript3/week3/ShoppingCart/shoppingcart-.js

```diff
@@ -0,0 +1,140 @@
+const body = document.getElementsByTagName("body");
+const h1 = document.getElementById("whos-shopping-cart");
+const itemList = document.getElementById("item-list");
+// const itemAddBtn = document.getElementById("add-btn");
+const cartTotal = document.getElementById("cart-total");
+const searchInput = document.getElementById("product-search-input");
+
+let cartTotalPrice = 0;
+
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+
+  convertToCurrency(currency) {
+    // Assuming dkr as default currency
+    try {
```

Comment:
you don't need a `try catch` block here because `fetch()` supports error handling.
see example : 
```
fetch('https://example.com/profile', {
  method: 'POST', // or 'PUT'
  headers: {
    'Content-Type': 'application/json',
  },
  body: JSON.stringify(data),
})
.then(response => response.json())
.then(data => {
  console.log('Success:', data);
})
.catch((error) => {
  console.error('Error:', error);
});
```